batch/batch.js

```diff
@@ -554,6 +554,28 @@ var canBatch = {
 		} else {
 			handler({});
 		}
+	},
+	/**
+	*/
+	debounce: function(handler){
```

Comment:
I think this needs to not be enumerable so that if `assign({}, canEvent)` is used, the object won't have a `debounce` method.  Also this needs to work on subsequent batches.  So if things change again, the `handler` will be called again.